docs/lms-openapi.yaml

```diff
@@ -2633,6 +2633,10 @@ paths:
                 date are returned. This is different from search_term because this filtering is done on
                 CourseOverview and not ElasticSearch.
 
+            keys (optional):
```

Comment:
NIT: We can probably rename this to something like `course_ids` or `course_keys` for the sake of consistency.